notes/stories/film/film-scifi-2020s.st.txt

```diff
@@ -791,6 +791,49 @@ murder [Zach was exposed for having deliberately electrocuted Richard Alling to
 peer pressure [Zach peer pressured some of the teens under his influence into committing wanton and callous acts of violence.]
 
 
+movie: Cosmic Sin (2021)
+========================
+
+:: Title
+Cosmic Sin
+
+:: Description
+Cosmic Sin is a 2021 American science fiction action film directed by Edward
+Drake and starring Bruce Willis and Frank Grillo.
+
+Synopsis: A disgraced former military commander is tasked back into active
+service to lead the fight against a hostile alien species that invades and
+conquers other worlds.
+
+:: Date
+2021-03-12
+
+:: References
+https://en.wikipedia.org/wiki/Cosmic_Sin
+
+:: Ratings
+1 <paul>
+
+:: Major Themes
+attack from outer space by a powerful conquering enemy [It was up to Ford and his team of elite soldiers to stop the malevolent Sigea aliens from conquering Earth.]
+extraterrestrial being [The malevolent Sigea aliens had designed on taking over the Earth.]
```

Comment:
I don't think you can have "designs on [verb phrase]". But you can have designs on Sweden, or the throne, for example.

Reply:
Rephrased.